Formula/rmw.rb

```diff
@@ -0,0 +1,26 @@
+class Rmw < Formula
+  desc "Safe-remove utility for the command-line"
+  homepage "https://remove-to-waste.info/"
+  url "https://github.com/theimpossibleastronaut/rmw/releases/download/v0.7.09/rmw-0.7.09.tar.gz"
+  sha256 "d0a85944d03e7ec2a94b25d6d6ac92880fd0d3f63d90bb9ed56b16418fd41c69"
+  license "GPL-3.0-or-later"
+  head "https://github.com/theimpossibleastronaut/rmw.git"
+
+  depends_on "ncurses"
+
+  def install
+    system  "./configure", "--enable-debug=no",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system  "make", "install"
+  end
+
+  test do
+    (testpath/"foo").write("bar")
+    system "#{bin}/rmw", (testpath/"foo")
+    system "#{bin}/rmw", "-u"
+    system "#{bin}/rmw", "-l"
+    system "#{bin}/rmw", "-vvg"
```

Comment:
Is it possible to compare some output here? Also, you could create a file, remove it and check that it doesn't exist.

Reply:
Thanks for reviewing @nandahkrishna . I added a check for the file existence like you suggested.

I could compare some output using substrings, except for one minor problem. I don't know ruby.

This is the output, which is all correct. Any suggestions on how I can compare output of those "system" commands by using substrings?

```
qandy@oceanus:~/src/homebrew-core/Formula$ brew test --verbose rmw.rb 
==> Testing rmw
==> /home/linuxbrew/.linuxbrew/Cellar/rmw/0.7.09/bin/rmw /tmp/rmw-test-20210513-22759-pusr7f/foo
Created directory /tmp/rmw-test-20210513-22759-pusr7f/.local/share/rmw
Created directory /tmp/rmw-test-20210513-22759-pusr7f/.config
Creating default configuration file:
  /tmp/rmw-test-20210513-22759-pusr7f/.config/rmwrc

Created directory /tmp/rmw-test-20210513-22759-pusr7f/.local/share/Waste/files/
Created directory /tmp/rmw-test-20210513-22759-pusr7f/.local/share/Waste/info/
1 file was removed to the waste folder
PASS (file not found)
==> /home/linuxbrew/.linuxbrew/Cellar/rmw/0.7.09/bin/rmw -u
+'/tmp/rmw-test-20210513-22759-pusr7f/.local/share/Waste/files/foo' -> '/tmp/rmw-test-20210513-22759-pusr7f/foo'
PASS (file exists again)
==> /home/linuxbrew/.linuxbrew/Cellar/rmw/0.7.09/bin/rmw -l
/tmp/rmw-test-20210513-22759-pusr7f/.local/share/Waste
==> /home/linuxbrew/.linuxbrew/Cellar/rmw/0.7.09/bin/rmw -vvg
PATH_MAX = 4096
RMW_FAKE_MEDIA_ROOT:false
config file: /tmp/rmw-test-20210513-22759-pusr7f/.config/rmwrc
purging is disabled ('purge_after' is set to '0')

most recent list (mrl file): /tmp/rmw-test-20210513-22759-pusr7f/.local/share/rmw/mrl
```

Reply:
> This is the output, which is all correct. Any suggestions on how I can compare output of those "system" commands by using substrings?
> 

@nandahkrishna ok, I figured out some ruby and pushed all my new changes. Thank you for the suggestions.